crawler.py

```diff
@@ -0,0 +1,91 @@
+""" Async Web Crawler """
+
+import asyncio
+import html
+import requests
+from re import findall, compile
+from requests_html import AsyncHTMLSession
+from typing import List
+
+
+async def get_html_content(url: str) -> AsyncHTMLSession.get:
```

Comment:
`AsyncHTMLSession.get` is not what you return here. You return a `Response` object. `AsyncHTMLSession.get` is a function.